chatbot/conversation_start.php

```diff
@@ -17,7 +17,7 @@
 $thisFile = __FILE__;
 
 /** @noinspection PhpIncludeInspection */
-require_once("../config/global_config.php");
+require_once(_CONF_PATH_ . 'global_config.php');
```

Comment:
Yes and no. `conversation_start.php` is called from other scripts like `gui/plain/index.php` where the config file is already included.
```php
require_once('../../config/global_config.php');
require_once('../../chatbot/conversation_start.php');
```
Otherwise it does not work. There no `config` folder in `gui` ;)